x-pack/plugins/security_solution/server/lib/detection_engine/routes/rules/create_rules_route.test.ts

```diff
@@ -55,21 +56,30 @@ describe.each([
 
   describe('status codes', () => {
     test('returns 200 with a rule created via RulesClient', async () => {
-      const response = await server.inject(getCreateRequest(), context);
+      const response = await server.inject(
+        getCreateRequest(),
+        requestContextMock.convertContext(context)
+      );
```

Comment:
Because lot of tests are mocking parts of the context. I tried it initially but it made the test even more awkward due to lines like

```ts
(await context.core).elasticsearch.client.mockReturnValue...
```

But I can use this approach if you prefer.

Reply:
Ah, I see! Ok, no worries, let's keep it as is 👍